src/api.rs

```diff
@@ -101,6 +101,14 @@ pub struct Api {
 impl Api {
     /// Returns a new `Api` element, and loads the MediaWiki site info from the `api_url` site.
     /// This is done both to get basic information about the site, and to test the API.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # tokio::runtime::Runtime::new().unwrap().block_on(async {
```

Comment:
Normally I write this as:
```
/// # async fn foo() -> {
/// let api = ...;
/// # }
```

That way we aren't dependent on starting a tokio runtime for doc tests. We can also return `Result<()>` so `?` syntax works rather than unwrap().